### PR TITLE
QuickStart: Reorganize and add generic KBS usage

### LINKED ISSUE #96: Quickstart guide no longer quick
It's great to see how committed people have been to updating the quickstart guide, but we are starting to have some organizational issues and in general the guide is becoming unwieldy both for people working on it and people reading it.

For starters we should probably split the quickstart into separate guides focusing on various aspects of the project. That said, let's make sure that we don't lose sight of the collective goals and interface of the project. 

It might also be time to adopt a more serious documentation framework. Many CNCF projects use some kind of documentation that transforms markdown into an actual website. Is this worth the effort for us at this time? How would we host such a site?

### LINKED ISSUE #101: Quick-Start: CoCo Key Broker System usage.
In v0.5.0, Generic KBS will be ready. We may need to briefly guide users in the Quick Start document on how to use Generic KBS to support pulling encrypted and signed images.

BTW, as mentioned in #96 . The current Quick Start is bloated. Maybe we should put the use of KBS that only supports specific TEE, such as `simple-kbs` and `EAA Verdictd`, in the lower level document and point to them with links, while leaving only the use of Generic KBS in Quick Start.

cc @fitzthum @wainersm

### PR DESCRIPTION
**Pull request by jialez0**
The first commit of this PR fixed: #96, it make the technology stacks for special HW separate markdown pages:
- Use simple-kbs to encrypt container image and deploy it on SEV: `guides/sev-guide.md`
- Use Verdictd to encrypt container image and deploy it on TDX: `guides/eaa-verdictd-guide.md`

The second commit of this PR fixed: #101, it add the usage of CoCo Key Broker System to `quickstart.md`

cc @Xynnn007 @sameo @fitzthum @wainersm @ryansavino

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -380,618 +380,173 @@ $ ssh -i ccv0-ssh root@$(kubectl get service ccv0-ssh -o jsonpath="{.spec.cluste
...
+        }
+    }
+}
 ```
 
-The above check will only work if the `encrypted-image-tests` guest launch is the only consuming qemu process running.
+Be sure to replace `[REGISTRY_URL]` with the desired registry URL of the encrypted image.
 
-Now, recalculate the SEV guest firmware measurement and store the `simple-kbs` policy in the database:
+Register the image pulling validation policy file to KBS storage:
```
Comment (Xynnn007) [inline quickstart.md:512]: We also need to register the verification key to KBS storage, too

File: quickstart.md [hunk 2]
```diff
@@ -380,618 +380,173 @@ $ ssh -i ccv0-ssh root@$(kubectl get service ccv0-ssh -o jsonpath="{.spec.cluste
...
-RUN ssh-keygen -t ed25519 -f /etc/ssh/ssh_host_ed25519_key -P ""
+To quick start the KBS cluster, a `docker-compose` yaml is provided to launch.
 
-# A password needs to be set for login to work. An empty password is
-# unproblematic as password-based login to root is not allowed.
-RUN passwd -d root
+```shell
+# Clone KBS git repository
+git clone https://github.com/confidential-containers/kbs.git
+cd kbs
```
Comment (Xynnn007) [inline quickstart.md:413]: Or we directly give the link to the `docker-compoer` here, as we do not the source code of kbs to deploy docker-compose
```suggestion
wget https://github.com/confidential-containers/kbs/blob/main/docker-compose.yml
```

**Review by Xynnn007 — COMMENTED** [review]
cc-kbc part LGTM. Great thanks for this!

File: quickstart.md (hunk 1 above)
Comment (jialez0) [reply quickstart.md:512]: Yes,  guide of this is at line 480 :-)

**Review by fitzthum — APPROVED** [review]
I have a number of comments, but I am planning to do a major overhaul of the quickstart guide for release 0.5.0, so I will implement them myself. The content added here looks good.

**Review by sameo — APPROVED** [review]

**Pull request merged**